### PR TITLE
lowercase anchor links in /Web/SVG

### PR DESCRIPTION
**Pull request by peterbe**
Partial (quick) solution to https://github.com/mdn/yari/issues/2524

### CODE REVIEW

**Comment by peterbe** [comment]
@Ryuno-Ki Did you get a notification on this PR? I'm just checking.

**Comment by Ryuno-Ki** [comment]
@peterbe Since I hit the join link only yesterday, I was now informed via your mention.
Looking into it.

**Comment by Ryuno-Ki** [comment]
Is it on purpose, that only the `href` were attribute without corresponding `id`s or `name`s? (I'm catching up with a couple hundred MDN/content notifications …)

**Comment by peterbe** [comment]
Basically, if you have this:
```html
<h2 id="Foo">Foo</a>

<a href="#Foo">Go to Foo</a>
```
becomes...
```html
<h2 id="foo">Foo</a>

<a href="#Foo">Go to Foo</a>
```
(note the lowercase `foo` in the `h2`)
when Yari builds. 
We'll add some code so the `href` is equally automatically treated. But we don't have that yet. This PR is about cleaning it up at the source.

File: files/en-us/web/svg/attribute/begin/index.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h2 id="discard">discard</h2>
  </tbody>
 </table>
 
-<p>The definition of <code>&lt;begin-value-list&gt;</code> is the <a href="#animate_animateColor_animateMotion_animateTransform_set">same as for the other animation elements</a>.</p>
+<p>The definition of <code>&lt;begin-value-list&gt;</code> is the <a href="#animate_animatecolor_animatemotion_animatetransform_set">same as for the other animation elements</a>.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/begin/index.html:91]: Confirmed working.

File: files/en-us/web/svg/attribute/conditional_processing/index.html [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
 <div class="index">
 <ul>
- <li><a href="#attr-externalResourcesRequired">externalResourcesRequired</a></li>
- <li><a href="#attr-requiredExtensions">requiredExtensions</a></li>
- <li><a href="#attr-requiredFeatures">requiredFeatures</a></li>
- <li><a href="#attr-systemLanguage">systemLanguage</a></li>
+ <li><a href="#attr-externalresourcesrequired">externalResourcesRequired</a></li>
+ <li><a href="#attr-requiredextensions">requiredExtensions</a></li>
+ <li><a href="#attr-requiredfeatures">requiredFeatures</a></li>
+ <li><a href="#attr-systemlanguage">systemLanguage</a></li>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/conditional_processing/index.html:18]: These don't work, as in, do not change focus. Checking the source code, I can see, that the target's are `<dt>`s.

Plus, the „Conditional Processing” Attribute is not linked from the SVG Attribute page (unrelated to this PR).

File: files/en-us/web/svg/attribute/d/index.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 <p>The <strong><code>d</code></strong> attribute defines a path to be drawn.</p>
 
-<p>A path definition is a list of <a href="#Path_commands">path commands</a> where each command is composed of a command letter and numbers that represent the command parameters. The commands are detailed below.</p>
+<p>A path definition is a list of <a href="#path_commands">path commands</a> where each command is composed of a command letter and numbers that represent the command parameters. The commands are detailed below.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/d/index.html:12]: Confirmed working.

File: files/en-us/web/svg/attribute/fill-opacity/index.html [hunk 4]
```diff
@@ -57,7 +57,7 @@ <h2 id="Usage_notes">Usage notes</h2>
  </tbody>
 </table>
 
-<p class="note"><strong>Note:</strong> SVG2 introduces percentage values for <code>fill-opacity</code>, however, it is not widely supported yet (<em>See <a href="#Browser_Compatibility">Browser compatibility</a> below</em>) as a consequence, it is best practices to set opacity with a value in the range <code>[0-1]</code>.</p>
+<p class="note"><strong>Note:</strong> SVG2 introduces percentage values for <code>fill-opacity</code>, however, it is not widely supported yet (<em>See <a href="#browser_compatibility">Browser compatibility</a> below</em>) as a consequence, it is best practices to set opacity with a value in the range <code>[0-1]</code>.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/fill-opacity/index.html:60]: Confirmed working.

Comment (Ryuno-Ki) [reply files/en-us/web/svg/attribute/fill-opacity/index.html:60]: This element is lacking the „notecard” class (unrelated to this PR)

File: files/en-us/web/svg/attribute/stroke-linejoin/index.html [hunk 5]
```diff
@@ -100,7 +100,7 @@ <h2 id="Usage_context">Usage context</h2>
 
 <h3 id="arcs">arcs</h3>
 
-<div class="notecard note"><strong>Note:</strong> the <code>arcs</code> value as been introduced in SVG2 and it isn't widely supported yet, see <a href="#Browser_compatibility">Browser compatibility</a> bellow for details.</div>
+<div class="notecard note"><strong>Note:</strong> the <code>arcs</code> value as been introduced in SVG2 and it isn't widely supported yet, see <a href="#browser_compatibility">Browser compatibility</a> bellow for details.</div>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/stroke-linejoin/index.html:103]: Confirmed working.

Comment (Ryuno-Ki) [reply files/en-us/web/svg/attribute/stroke-linejoin/index.html:103]: Typo: bellow should be below (unrelated to this PR)

File: files/en-us/web/svg/attribute/stroke-opacity/index.html [hunk 6]
```diff
@@ -57,7 +57,7 @@ <h2 id="Usage_notes">Usage notes</h2>
  </tbody>
 </table>
 
-<p class="note"><strong>Note:</strong> SVG2 introduces percentage values for <code>stroke-opacity</code>, however, it is not widely supported yet (<em>See <a href="#Browser_Compatibility">Browser compatibility</a> below</em>) as a consequence, it is best practices to set opacity with a value in the range <code>[0-1]</code>.</p>
+<p class="note"><strong>Note:</strong> SVG2 introduces percentage values for <code>stroke-opacity</code>, however, it is not widely supported yet (<em>See <a href="#browser_compatibility">Browser compatibility</a> below</em>) as a consequence, it is best practices to set opacity with a value in the range <code>[0-1]</code>.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/attribute/stroke-opacity/index.html:60]: Confirmed working.

Comment (Ryuno-Ki) [reply files/en-us/web/svg/attribute/stroke-opacity/index.html:60]: This paragraph is lacking the „notecard” class (unrelated to this PR)

File: files/en-us/web/svg/content_type/index.html [hunk 7]
```diff
@@ -389,6 +389,6 @@ <h2 id="URL">URL</h2>
 
 	<p>A URL is a sequence of {{Glossary("Unicode")}} characters, building an address to an internal or external resource.</p>
 
-	<p>Before SVG 2, the more limited <a href="#IRI">IRI</a> content type was used instead, because the URL specification was not standardized before.</p>
+	<p>Before SVG 2, the more limited <a href="#iri">IRI</a> content type was used instead, because the URL specification was not standardized before.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/content_type/index.html:392]: Confirmed working.

Comment (Ryuno-Ki) [reply files/en-us/web/svg/content_type/index.html:392]: This page seem not to be linked from the SVG page (unrelated to this PR)

File: files/en-us/web/svg/element/use/index.html [hunk 8]
```diff
@@ -40,7 +40,7 @@
 <p>For security reasons, browsers may apply the <a href="/en-US/docs/Web/Security/Same-origin_policy">same-origin policy</a> on <code>use</code> elements and may refuse to load a cross-origin URL in the {{SVGAttr("href")}} attribute. There is currently no defined way to set a cross-origin policy for <code>use</code> elements.</p>
 
 <div class="warning">
-<p>Since SVG 2, the {{SVGAttr("xlink:href")}} attribute is deprecated in favor of {{SVGAttr("href")}}. See {{SVGAttr("xlink:href")}} page for more information. However, {{SVGAttr("xlink:href")}} can still be required in practice for cross-browser compatibility (see the <a href="#Browser_compatibility">compatibility table</a> below).</p>
+<p>Since SVG 2, the {{SVGAttr("xlink:href")}} attribute is deprecated in favor of {{SVGAttr("href")}}. See {{SVGAttr("xlink:href")}} page for more information. However, {{SVGAttr("xlink:href")}} can still be required in practice for cross-browser compatibility (see the <a href="#browser_compatibility">compatibility table</a> below).</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/element/use/index.html:43]: Confirmed working.

**Review by Ryuno-Ki — CHANGES_REQUESTED** [review]
I checked out your branch locally.
If you can repair the „conditional processing” page (or back out the changes) I can approve this PR.

There are some unrelated comments you might want to pick up on an update.

File: files/en-us/web/svg/attribute/conditional_processing/index.html (hunk 2 above)
Comment (peterbe) [reply files/en-us/web/svg/attribute/conditional_processing/index.html:18]: Fixed now.

Comment (Ryuno-Ki) [reply files/en-us/web/svg/attribute/conditional_processing/index.html:18]: Confirmed :)

**Review by Ryuno-Ki — APPROVED** [review]
Works like a charm now!

**Pull request merged**